src/server/scripts/Spells/spell_warlock.cpp

```diff
@@ -284,6 +285,10 @@ class spell_warl_generic_scaling : public AuraScript
             SpellSchoolMask schoolMask = SpellSchoolMask(aurEff->GetSpellInfo()->Effects[aurEff->GetEffIndex()].MiscValue);
             int32 modifier = schoolMask == SPELL_SCHOOL_MASK_NORMAL ? 35 : 40;
             amount = CalculatePct(std::max<int32>(0, owner->GetResistance(schoolMask)), modifier);
+            if (owner->HasAura(SPELL_WARLOCK_PET_VOID_STAR_TALISMAN) && schoolMask != SPELL_SCHOOL_MASK_NORMAL)
+            {
+                amount += 130;
```

Comment:
just want to know where it is from

Reply:
https://www.wowhead.com/wotlk/item=30449/void-star-talisman

Reply:
Hot